modules/apps/map/map-openstreetmap/src/main/resources/META-INF/resources/js/MapOpenStreetMap.es.js

```diff
@@ -147,22 +155,5 @@ MapOpenStreetMap.POSITION_MAP = {
 	[MapBase.POSITION.TOP_RIGHT]: 'topright',
 };
 
-/**
- * State definition.
- * @type {!Object}
- * @static
- */
-MapOpenStreetMap.STATE = {
-	...MapBase.STATE,
-
-	/**
-	 * Url used for fetching map tile information
-	 * @type {string}
-	 */
-	tileURI: Config.string().value(
-		'//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
-	),
```

Comment:
Because this is `STATE` originally, we need to make sure and re-implement the get/set for this value.